README.md

```diff
@@ -3,3 +3,13 @@
 [![pre-commit hooks](https://github.com/AlexMomot-717/python-project-83/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/AlexMomot-717/python-project-83/actions)
 [![Maintainability](https://api.codeclimate.com/v1/badges/7ad5b8e6d1a22a4fdf34/maintainability)](https://codeclimate.com/github/AlexMomot-717/python-project-83/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/7ad5b8e6d1a22a4fdf34/test_coverage)](https://codeclimate.com/github/AlexMomot-717/python-project-83/test_coverage)
+
+### Building and running your application
+
+When you're ready, start application by running:
+
+```
+docker compose up --build
+```
```

Comment:
let's use make here (as we have targets already) to explain how to start the application

part with "when you're ready ..." is not needed